plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -5,6 +5,17 @@
                 xmlns:dita2html="http://dita-ot.sourceforge.net/ns/200801/dita2html"
                 exclude-result-prefixes="#all"
                 version="2.0">
+
+    <xsl:param name="using-input"/>
+    <xsl:param name="includes-pdf"/>
+
+    <xsl:variable name="input-string" select="$using-input"/>
+    <xsl:variable name="map-name"
+                  select="tokenize($using-input, '/')[last()]"/>
+    <xsl:variable name="publication-name"
+                  select="substring-before($map-name, '.ditamap')"/>
```

Comment:
We don't know which exactly extension we have here. 
You don't need this part of code at all just because you already get input file without extension in the ant. So, please use 1 parameter instead of 3 variables.